### ISSUE TITLE
SA1008 false positive for operators

### ISSUE THREAD

**Issue by distantcam** [DotNetAnalyzers/StyleCopAnalyzers#2146]
When defining an operator SA1008 falsely reports that the space before the parenthesis should be removed in the following example

```csharp
public class MyInt {
    public int Value;

    public static implicit operator int (MyInt mi) => mi.Value; // This line has the SA1008 warning
}

**Comment by distantcam** [comment]
Nevermind my code formatter was fighting with the StyleCopAnalyzer.

**Comment by sm-g** [comment]
seems it is default formatter behaviour on VS 15.4.0, also see https://github.com/dotnet/roslyn/issues/4240
